docs/pages/access-controls/guides/device-trust.mdx

```diff
@@ -137,17 +152,55 @@ Relogin to fetch updated certificates:
 $ tsh logout; tsh login --proxy=example.com --user=alice
 ```
 
-Having the necessary permissions, let's add a device to the inventory. Replace
-the `$SERIAL` below with your macOS device serial number (visible under Apple ->
-"About This Mac" -> "Serial number", or use the command below).
+Before we can enroll the device, we need to register it. In order to register
+a device, we need to first determine its serial number.
+
+<Tabs>
+  <TabItem label="macOS">
+    The serial number of a macOS device can be determined in two ways.
+
+    Through the UI, visible under Apple -> "About This Mac" -> "Serial number".
+
+    Through the terminal:
+    ```code
+    $ ioreg -c IOPlatformExpertDevice -d 2 | grep -i IOPlatformSerialNumber | awk -F'"' '{print $4}'
+    (=devicetrust.asset_tag=)
+    ```
+  </TabItem>
+
+  <TabItem label="Windows">
+    Determining the serial number for a device running Windows is a little more
+    complicated than on macOS. This is because Windows devices can have multiple
+    serial numbers depending on the configuration made by the manufacturer.
+
+    Teleport will pick the first available value from the following:
+
+    - System asset tag
+    - System serial number
+    - Baseboard serial number
+
+    To find the value chosen by Teleport, run the following command:
+
+    ```code
+    $ tsh device collect
```

Comment:
Was `tsh device collect` also added to the CLI reference doc?

Reply:
Mm, not sure, this wasn't added as part of the TPM work.

Reply:
I have mentioned hidden commands in docs before, like the "diag" commands, without necessarily adding them to the CLI reference. My 2c is that I wouldn't add this one, but your call.

Reply:
We could also expose a simpler, `tsh device serial-number` command, that spits out the serial number and nothing else. Then we could write a single section here.

Reply:
I like the creation of `tsh device serial-number` command - and it'll be easy enough for us to do. I suggest we merge these docs as is, and then do that command as a separate exercise though.

Reply:
Sounds good to me.